src/pages/Thanks/ThanksPage.vue

```diff
@@ -30,6 +31,7 @@
 				:receipt="receipt"
 				:badges-achieved="badgesAchieved"
 				:router="$router"
+				:guest-username="guestUsername"
```

Comment:
I think we can just focus on the new single TY page, so we could rollback these old components to simplify this changeset.

Reply:
This work is already done, said that, if for some reason, product feedback or whatever, we need to turn off single version this feature stills working in other cases... Otherwise this we'll easy to remove if single version is a winner having a clean up of the page in the future.